### ISSUE TITLE
Modules are missing when installing the package

### ISSUE THREAD

**Issue by sajith** [atlanticwave-sdx/datamodel#43]
Here's a small test:

```console
$ mkdir /tmp/datamodel/
$ cd /tmp/datamodel/
$ python3.9 -m venv venv
$ source venv/bin/activate
$ pip install git+https://github.com/atlanticwave-sdx/datamodel.git
Defaulting to user installation because normal site-packages is not writeable
Collecting git+https://github.com/atlanticwave-sdx/datamodel.git
  Cloning https://github.com/atlanticwave-sdx/datamodel.git to /tmp/pip-req-build-wfiuoycn
  Running command git clone --filter=blob:none -q https://github.com/atlanticwave-sdx/datamodel.git /tmp/pip-req-build-wfiuoycn
  Resolved https://github.com/atlanticwave-sdx/datamodel.git to commit 3a83c58fb81ca8af496d9946297544727399ea45
  Preparing metadata (setup.py) ... done
Using legacy 'setup.py install' for sdxdatamodel, since package 'wheel' is not installed.
Installing collected packages: sdxdatamodel
    Running setup.py install for sdxdatamodel ... done
Successfully installed sdxdatamodel-0.0.1
```

Although datamodel could be installed from git sources, the installed package is not usable:

```console
$ python
Python 3.10.6 (main, Aug  2 2022, 00:00:00) [GCC 12.1.1 20220507 (Red Hat 12.1.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from sdxdatamodel.models.node import Node
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'sdxdatamodel.models'
>>> 
```
Turns out that installing datamodel installed an empty package, without the modules that it meant to contain.

```console
tree venv/lib/python3.9/site-packages/sdxdatamodel
venv/lib/python3.9/site-packages/sdxdatamodel
├── __init__.py
└── __pycache__
    └── __init__.cpython-39.pyc

1 directory, 2 files
```

**Comment by sajith** [comment]
Installing datamodel directly from a source tree (not from a git repository, as shown above), seems to work:

```console
$ pip install ~/projects/atlanticwave-sdx/datamodel/
Processing /home/sajith/projects/atlanticwave-sdx/datamodel
  Preparing metadata (setup.py) ... done
Using legacy 'setup.py install' for sdxdatamodel, since package 'wheel' is not installed.
Installing collected packages: sdxdatamodel
  Attempting uninstall: sdxdatamodel
    Found existing installation: sdxdatamodel 0.0.1
    Uninstalling sdxdatamodel-0.0.1:
      Successfully uninstalled sdxdatamodel-0.0.1
    Running setup.py install for sdxdatamodel ... done
Successfully installed sdxdatamodel-0.0.1
```

The installed library doesn't seem to be so empty now:

```console
$ tree venv/lib/python3.9/site-packages/sdxdatamodel
venv/lib/python3.9/site-packages/sdxdatamodel
├── __init__.py
├── models
│   ├── connection.py
│   ├── __init__.py
│   ├── link_measurement_period.py
│   ├── link.py
│   ├── location.py
│   ├── node.py
│   ├── port.py
│   ├── __pycache__
│   │   ├── connection.cpython-39.pyc
│   │   ├── __init__.cpython-39.pyc
│   │   ├── link.cpython-39.pyc
│   │   ├── link_measurement_period.cpython-39.pyc
│   │   ├── location.cpython-39.pyc
│   │   ├── node.cpython-39.pyc
│   │   ├── port.cpython-39.pyc
│   │   ├── service.cpython-39.pyc
│   │   └── topology.cpython-39.pyc
│   ├── service.py
│   └── topology.py
├── parsing
│   ├── connectionhandler.py
│   ├── exceptions.py
│   ├── __init__.py
│   ├── linkhandler.py
│   ├── locationhandler.py
│   ├── nodehandler.py
│   ├── porthandler.py
│   ├── __pycache__
│   │   ├── connectionhandler.cpython-39.pyc
│   │   ├── exceptions.cpython-39.pyc
│   │   ├── __init__.cpython-39.pyc
│   │   ├── linkhandler.cpython-39.pyc
│   │   ├── locationhandler.cpython-39.pyc
│   │   ├── nodehandler.cpython-39.pyc
│   │   ├── porthandler.cpython-39.pyc
│   │   ├── servicehandler.cpython-39.pyc
│   │   └── topologyhandler.cpython-39.pyc
│   ├── servicehandler.py
│   └── topologyhandler.py
├── __pycache__
│   └── __init__.cpython-39.pyc
├── topologymanager
│   ├── grenmlconverter.py
│   ├── __init__.py
│   ├── manager.py
│   ├── __pycache__
│   │   ├── grenmlconverter.cpython-39.pyc
│   │   ├── __init__.cpython-39.pyc
│   │   ├── manager.cpython-39.pyc
│   │   └── temanager.cpython-39.pyc
│   └── temanager.py
└── validation
    ├── connectionvalidator.py
    ├── __init__.py
    ├── __pycache__
    │   ├── connectionvalidator.cpython-39.pyc
    │   ├── __init__.cpython-39.pyc
    │   └── topologyvalidator.cpython-39.pyc
    └── topologyvalidator.py

9 directories, 52 files
```

**Comment by sajith** [comment]
Seems that this line is to blame:

https://github.com/atlanticwave-sdx/datamodel/blob/3a83c58fb81ca8af496d9946297544727399ea45/setup.cfg#L38

Removing that `include = sdxdatamodel` line seems to include all the required modules in the package build.

**Comment by sajith** [comment]
Fixed in PR #44.
